src/namespaces/apoc/date.ts

```diff
@@ -32,14 +33,10 @@ import { CypherFunction } from "../../expressions/functions/CypherFunctions";
  * )
  *```
  */
-export function convertFormat(
```

Comment:
I'm not sure if the formatting change here was a result of the line length change, or just my editor.